packages/client-core/src/admin/services/InstanceService.ts

```diff
@@ -52,10 +48,10 @@ export const useInstanceState = () => useState(state) as any as typeof state
 
 //Service
 export const InstanceService = {
-  fetchAdminInstances: async (incDec?: 'increment' | 'decrement', search: string = '') => {
+  fetchAdminInstances: async (incDec?: 'increment' | 'decrement', search: string | null = null) => {
     const dispatch = useDispatch()
     {
-      const skip = accessInstanceState().skip.value
+      const skip = 0
```

Comment:
Aren't we going to need skip to change in order to paginate?

Reply:
Let me change it back, I was just testing